examples/polyglot-zipkin/README.adoc

```diff
@@ -11,22 +11,26 @@ of the endpoints might not work as they are calling external services. Feel free
 services.
 
 == Run
+Before running ensure that containers from previous run were removed.
+
 [source,shell]
 ----
+$ export KAFKA_ZOOKEEPER=172.17.0.1
```

Comment:
Isn't this confusing the env variables required by the app, and those required by zipkin/apm servers?